packages/mermaid/src/diagrams/sankey/sankeyDB.ts

```diff
@@ -0,0 +1,101 @@
+import * as configApi from '../../config.js';
+import common from '../common/common.js';
+import {
+  setAccTitle,
+  getAccTitle,
+  getAccDescription,
+  setAccDescription,
+  setDiagramTitle,
+  getDiagramTitle,
+  clear as commonClear,
+} from '../../commonDb.js';
+
+// Variables where graph data is stored
+// Sankey diagram represented by nodes and links between those nodes
+// We have to track nodes uniqueness (by ID), thats why we need a mapping also
+//
+let links: SankeyLink[] = [];
+let nodes: SankeyNode[] = [];
+let nodesMap: Record<string, SankeyNode> = {};
+let nodeAlign = 'justify';
+
+const setNodeAlign = (alignment: string): void => {
+  const nodeAlignments: Set<string> = new Set(['left', 'right', 'center', 'justify']);
```

Comment:
Nit: You can move this outside the function to avoid re-initializing on each call.

Reply:
This function removed completely, node alignment is a config option now